src/pages/nf_version.ts

```diff
@@ -0,0 +1,36 @@
+import type { APIRoute } from 'astro';
+
+export const GET: APIRoute = async ({ params, request }) => {
+    try {
+        // get latest stable nextflow version from github releases
+        const response = await fetch('https://api.github.com/repos/nextflow-io/nextflow/releases');
+        if (!response.ok) {
+            throw new Error('Failed to fetch latest version');
+        }
```

Comment:
This is a static endpoint, right? So this error will break the build, but the live site will be unaffected?